test/models/report_test.rb

```diff
@@ -3,7 +3,19 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
+    @alice = users(:alice)
+    @bob = users(:bob)
+    @report = reports(:report)
+  end
+
+  test '#editable?' do
+    assert @report.editable?(@alice)
+    assert_not @report.editable?(@bob)
+  end
+
+  test '#created_on' do
+    assert_not_equal DateTime.tomorrow.to_date, @report.created_on
+    assert_equal DateTime.now.to_date, @report.created_on
```

Comment:
```suggestion
    assert_equal Date.today, @report.created_on
```

の方がシンプルですね。

ちなみにDateTimeは非推奨クラスです。

https://docs.ruby-lang.org/ja/latest/class/DateTime.html

あと、実はDate.todayよりもDate.currentの方がお勧めだったりします。
詳しい話はこちらに書いていますー。

[RubyとRailsにおけるTime, Date, DateTime, TimeWithZoneの違い \- Qiita](https://qiita.com/jnchito/items/cae89ee43c30f5d6fa2c)

Reply:
非推奨気づかすでした。。
ご指摘ありがとうございます！